docs/examples.rst

```diff
@@ -141,3 +141,14 @@ One form of running this example could be:
 .. explicit external hyperlink targets
 
 .. _examples: https://github.com/LLNL/UnifyFS/tree/dev/examples/src
+
+Transfer API 
+------------
+
+UnifyFS has a transfer API to move files from the UnifyFS file space out to external storage (or from external storage into the UnifyFS file space).  This can be invoked using the function API by linking, or using the available "transfer" application availalbe in the examples directory.  The example code also serves as a template for using the C function API. 
```

Comment:
typo s/availalbe/available/

Also, Travis is complaining about the trailing whitespace on lines 145 an 148

Reply:
also, you have redundant "available"s, get rid of the one before "transfer"